src/components/Movie/MovieInfo.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+MovieInfoLists = ({list, style, id}) => {
+  return (
+    <View
+      style={{
+        flex: 1,
+        flexDirection: 'row',
+        flexWrap: 'wrap',
+      }}>
+      {list.map((item, i) => {
+        return (
+          <Text
+            key={id == 0 ? item.id : id == 1 ? item.iso_3166_1 : item.iso_639_1}
```

Comment:
can't we use 'i' here directly, as that would be unique for every iteration? Also, instead of i, prefer 'index'.